src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py

```diff
@@ -143,6 +144,12 @@ def payload_to_semantic_attributes(
         if event_type is CBEventType.LLM:
             if model_name := serialized.get("model"):
                 attributes[LLM_MODEL_NAME] = model_name
+                attributes[LLM_INVOCATION_PARAMETERS] = {
+                    "model": model_name,
+                    "temperature": serialized["temperature"],
+                    "max_tokens": serialized["max_tokens"],
+                    **serialized["additional_kwargs"],
+                }
```

Comment:
Are we using dictionaries or JSON strings for unstructured objects?

Reply:
both works. but it looks like frontend is expecting a string, so i've changed it to a string. good catch.